### DISCUSSION TITLE
The number of followers on the profile page does not change.

### DISCUSSION THREAD

**Discussion by Erykalin1986** [community/community#63023] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The number of followers on the profile page does not change.

**Comment by 4chref** [comment]
there is no bug i have the same problem im currently live in Russia and i think its a problem of some internet restrictions when ever i use VPN every thing work normally you can check my profile and see by your self i added follower counter follow me and check again

**Reply by Bobstar55** [reply]
f4e90cd4e95a75b496ec4a643e71fe4f047af46e

**Comment by bxbdbdbj** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> The number of followers on the profile page does not change.

**Comment by Cjuan4631** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> The number of followers on the profile page does not change.
